src/Ubidots.cpp

```diff
@@ -26,60 +26,40 @@ Modified by Jose Garcia for Ubidots Inc
 */
 
 #include "Ubidots.h"
+
+/***************************************************************************
+CONSTRUCTOR
+***************************************************************************/
+
 /**
  * Constructor.
  * Default method is UDP
  * Default dsNmae is Particle
  */
+
 Ubidots::Ubidots(char* token, char* server) {
     _token = token;
     _server = server;
     _method = TYPE_UDP;
-    _dsName = "Particle";
-    lastValue = NULL;
-    currentValue = 0;
+    _dsName = "particle";
+    _currentValue = 0;
     val = (Value *)malloc(MAX_VALUES*sizeof(Value));
-    String str = Particle.deviceID();
+    String str = System.deviceID();
     _pId = new char[str.length() + 1];
-    strcpy(_pId, str.c_str());
-}
-/** 
- * This function is to set UDP or TCP method
- * @arg method is the method that you want to use
- * @return true uppon succes
- */
-void Ubidots::setMethod(uint8_t method) {
-    if (method >= 0 && method <= 2) {
-        _method = method;
-    }
+    strcpy(_dsName, str.c_str());
```

Comment:
it should be the label not the name the one that should have the device ID. Fixing for next commit.